src/jdk.jdi/share/classes/com/sun/tools/example/debug/tty/TTYResources.java

```diff
@@ -476,7 +476,8 @@ public Object[][] getContents() {
              "    -connect <connector-name>:<name1>=<value1>,...\n" +
              "                      connect to target VM using named connector with listed argument values\n" +
              "    -dbgtrace [flags] print info for debugging {0}\n" +
-             "    -trackvthreads    track virtual threads as they are created\n" +
+             "    -trackallthreads  Track all threads, including virtual threads.\n" +
```

Comment:
should this start from lower case letter for consistency?

Reply:
I debated which was better: start lower to be consistent with other help lines, or start with upper since there would be two sentences. I went with the latter.